rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    person.add_rental(self)
+    @book = book
+    book.add_rental(self)
```

Comment:
- [ ] Instead of `person.add_rental(self)` or `book.add_rental(self)` now you can use `person.rentals << self` and `book.rentals << self` as now we're initializing in `add_rental`.

Reply:
Thanks very much, @Deepakdanger for the review, I have really learnt a lot.